### PR TITLE
[CI] Fixed Jenkins artifacts

### PR DESCRIPTION
**Pull request by mryzhov**
set CPACK_COMPONENTS_ALL

### CODE REVIEW

**Review by Wovchena — APPROVED** [review]
Note, these aren't all tokenizers problems. There's
```cmake
install(FILES "${openvino_tokenizers_SOURCE_DIR}/LICENSE"
              "${openvino_tokenizers_SOURCE_DIR}/third-party-programs.txt"
              "${openvino_tokenizers_SOURCE_DIR}/SECURITY.md"
        DESTINATION "${PY_BUILD_CMAKE_PACKAGE_NAME}-${PY_BUILD_CMAKE_PACKAGE_VERSION}.dist-info"
        COMPONENT openvino_tokenizers_licenses
        EXCLUDE_FROM_ALL)
```
which doesn't expect that it's going to be installed as part of an archive.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -382,6 +382,8 @@ install(FILES "${openvino_tokenizers_BINARY_DIR}/python/__version__.py"
 # Cpack configuration
 #
 
+set(CPACK_COMPONENTS_ALL openvino_tokenizers_python openvino_tokenizers openvino_tokenizers_licenses)
```
Comment (mryzhov) [inline src/CMakeLists.txt:385]:
```suggestion
set(CPACK_COMPONENTS_ALL openvino_tokenizers_python openvino_tokenizers)
```

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -382,6 +382,8 @@ install(FILES "${openvino_tokenizers_BINARY_DIR}/python/__version__.py"
 # Cpack configuration
 #
 
+set(CPACK_COMPONENTS_ALL openvino_tokenizers_python openvino_tokenizers)
```
Comment (mryzhov) [inline src/CMakeLists.txt:385]:
```suggestion
set(CPACK_COMPONENTS_ALL openvino_tokenizers)
```

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -382,6 +382,8 @@ install(FILES "${openvino_tokenizers_BINARY_DIR}/python/__version__.py"
 # Cpack configuration
 #
 
+set(CPACK_COMPONENTS_ALL openvino_tokenizers)
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply src/CMakeLists.txt:385]: That how it was in previous release. @apaniukov have you aligned that?

[the remark this answers is not in the document]
Comment (apaniukov) [reply src/CMakeLists.txt:385]: The previous release contains only the `runtime` dir: https://storage.openvinotoolkit.org/repositories/openvino_tokenizers/packages/2024.2.0.0/

[the remark this answers is not in the document]
Comment (Wovchena) [reply src/CMakeLists.txt:385]: > The previous release contains only the `runtime` dir: https://storage.openvinotoolkit.org/repositories/openvino_tokenizers/packages/2024.2.0.0/

There are no doubts about that :)

[the remark this answers is not in the document]
Comment (mryzhov) [reply src/CMakeLists.txt:385]: Talked to @jacekpawlak and agreed to add the following files inside docs/openvino_tokenizers folder:
license, 3rd-party.txt, readme

**Comment by Wovchena** [comment]
Aren't these names occupied by OpenVINO? GenAI renames its file: https://github.com/openvinotoolkit/openvino.genai/blob/f0682f95916f5b80f2112bc69e17bde4a420ce0e/CMakeLists.txt#L49

File: src/CMakeLists.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Wovchena) [reply src/CMakeLists.txt]: Aren't these names occupied by OpenVINO? GenAI renames its file: https://github.com/openvinotoolkit/openvino.genai/blob/f0682f95916f5b80f2112bc69e17bde4a420ce0e/CMakeLists.txt#L49

**Review by jacekpawlak — APPROVED** [review]

File: src/CMakeLists.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (mryzhov) [reply src/CMakeLists.txt]: that is why we decided to save them inside separate subfolder

**Pull request merged**